content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>I think we can drop the explanation of <code>np.hypot()</code>  and link to the documentation instead. However, an explanation or link to say why we normalize the image back to values between 0 and 255 would be nice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
- Added links to `np.hypot()` and `scipy.ndimage` filters (Fourier Gaussian and Sobel) 👍 
- Added a detailed explanation with links to image normalization 👍 

Thanks 😃